crates/sui-types/src/messages.rs

```diff
@@ -1959,6 +1957,9 @@ pub enum ExecutionFailureStatus {
     MoveAbort(MoveLocation, u64), // TODO func def + offset?
     VMVerificationOrDeserializationError,
     VMInvariantViolation,
+
+    /// The total amount of coins to be paid is larger than the maximum value of u64.
+    TotalAmountOverflow,
```

Comment:
Could you also add a comment on 1963 like "if you want to add new enum, please add it at the end for backward compatibility"?